Index: tools/tinyos/misc/tos-locate-jre
===================================================================
--- tinyos-2.1.1.3/tools/tinyos/misc/tos-locate-jre	(revision 5720)
+++ build-tinyos-tools/tools/tinyos/misc/tos-locate-jre	(working copy)
@@ -51,7 +51,7 @@
 
     Darwin)
     #Just statically typed in, uses default location of installation for XTools.  May need to be fixed
-    jhome=/System/Library/Frameworks/JavaVM.framework/Versions/CurrentJDK
+    jhome=/System/Library/Frameworks/JavaVM.framework/Versions/Current
     ;;
 
     Linux)
